plugins/woocommerce/client/admin/client/wp-admin-scripts/email-editor-integration/email_status.tsx

```diff
@@ -0,0 +1,156 @@
+/**
+ * External dependencies
+ */
+import { __, sprintf } from '@wordpress/i18n';
+import { select, dispatch } from '@wordpress/data';
+import {
+	PanelRow,
+	Button,
+	Flex,
+	FlexItem,
+	Dropdown,
+	RadioControl,
+} from '@wordpress/components';
+import { closeSmall } from '@wordpress/icons';
+import { store as coreDataStore, useEntityProp } from '@wordpress/core-data';
+
+/**
+ * Internal dependencies
+ */
+import { EMAIL_STATUSES } from '../../settings-email/settings-email-listing-status';
+
+interface EmailStatusProps {
+	className?: string;
+}
+
+/**
+ * EmailStatus component displays and allows changing the status of a transactional email.
+ * It shows whether the email is enabled, disabled, or manually sent.
+ *
+ * @param {EmailStatusProps} props - Component props.
+ * @return {JSX.Element} Rendered component.
+ */
+export function EmailStatus( { className }: EmailStatusProps ): JSX.Element {
+	const [ woocommerce_email_data ] = useEntityProp(
+		'postType',
+		'woo_email',
+		'woocommerce_data'
+	);
+
+	const isManual = woocommerce_email_data?.is_manual;
+	let statusValue = 'disabled';
```

Comment:
For consistency with the `WC_Email` object, we can set the default status to `'enabled'`, although the `enabled` property should always be present in `woocommerce_email_data`.